### ISSUE TITLE
What was the dataset that you used to train this ControlNet?

### ISSUE THREAD

**Issue by gdivya304** [Georgefwt/Face-Landmark-ControlNet#7]
I want to take this further and implement something similar, what dataset did you use for training this model?

**Comment by Georgefwt** [comment]
I'm glad to hear that. I used CelebA-HQ-img and FFHQ for training. Actually, a better approach would be to use street scene datasets for training, because the faces in these two datasets have already been aligned.

**Comment by gdivya304** [comment]
Also did you use images + captions for training? Like in the basic example for controlnet with the circle, they used colour captions with source and target images. In this, the source would be the landmarks data? With the ffhq or celebA as target?

**Comment by Georgefwt** [comment]
You can use GPT to generate corresponding captions, for example like 'A photo of a man'. You can use GPT to generate many similar captions for training.

**Comment by gdivya304** [comment]
Just checking if you have some idea about this, but would it be possible to combine this with a lora model finetuning of stable diffusion? Or would it require some architecture change?

**Comment by gdivya304** [comment]
@Georgefwt Would it be possible to combine this controlnet with a lora model?

**Comment by Georgefwt** [comment]
Of course, this ControlNet has no structural differences from other ControlNets, so LoRA can be used.
